public/locales/en/panel-time.json

```diff
@@ -0,0 +1,68 @@
+{
+  "select-time-title": "Select Time",
+  "error": {
+    "outside-range": "Can't interact with dates outside the range April 20, 271821 BC to Sep 13, 275760 AD."
+  },
+  "simulation-speed-title": "Simulation Speed",
+  "realtime": "Realtime",
+  "now": "Now",
+  "time-parts": {
+    "Milliseconds_one": "Milliseconds",
+    "Milliseconds_other": "Milliseconds",
+    "Seconds_one": "Second",
+    "Seconds_other": "Seconds",
+    "Minutes_one": "Minute",
+    "Minutes_other": "Minutes",
+    "Hours_one": "Hour",
+    "Hours_other": "Hours",
+    "Days_one": "Day",
+    "Days_other": "Days",
+    "Weeks_one": "Week",
+    "Weeks_other": "Weeks",
+    "Months_one": "Month",
+    "Months_other": "Months",
+    "Years_one": "Year",
+    "Years_other": "Years"
```

Comment:
these should not start with a capitla letter

Reply:
for example: 
```javascript
t(`time-parts.${TimePart.Seconds}`, {count: 1}) 
```